euexit/templates/euexit/international-contact-form.html

```diff
@@ -1,28 +1,35 @@
-{% extends "core/base.html" %}
+{% extends "core/base_international.html" %}
 
 {% load static from staticfiles %}
 {% load render_form from directory_components_tags %}
-
-{% block sharing_metadata %}{% endblock %}
+{% load add_export_elements_classes from directory_components_tags %}
 
 {% block head_css %}
     {{ block.super }}
+    <link href="{% static 'styles/euexit-form.css' %}" rel="stylesheet" type="text/css" />
 {% endblock %}
 
+
 {% block content %}
     <section>
         <div class="container">
-            <div class="grid-row">
-                <div class="column-one-half">
-                    <form method="post">
+            {% include 'euexit/breadcrumbs.html' %}
+            <form method="post">
+                <div class="grid-row">
+                    <div class="column-one-half">
                         <div>
-                            <h3 class="heading-medium">EU Exit help</h3>
-                            <span class="font-xsmall">If you don't find an anwswer...</span>
+                            <h1 class="heading-xlarge">{{ page.heading }}</h3>
+                            <p>{{ page.body_text|add_export_elements_classes }}</p>
                         </div>
+                    </div>
+                </div>
+                <div class="grid-row">
```

Comment:
you can get rid of `grid-row` and `column-` divs here because the form widgets do this on their own